api/admin/controller/__init__.py

```diff
@@ -2266,7 +2277,7 @@ def search_field_values(self) -> dict:
     # 1 hour in-memory cache
     @memoize(ttls=3600)
     def _search_field_values_cached(self, collection_ids: List[int]) -> dict:
-        def _unzip(values: List[tuple]) -> dict:
+        def _unzip(values: List[Tuple[str, int]]) -> dict:
```

Comment:
Just curious:  is there any reason to have the  _unzip function defined within _search_field_values_cached?  I'm not sure how the python interpreter works on this level, but I'm wondering if it would be more clear from a readability as well as code execution point of view  to have this method moved up to the class level.